src/main/java/fr/insalyon/creatis/gasw/GaswConfiguration.java

```diff
@@ -144,6 +148,7 @@ private void loadConfigurationFile() throws GaswException {
         try {
             executionPath = new File("").getAbsolutePath();
             simulationID = executionPath.substring(executionPath.lastIndexOf("/") + 1);
+            workflowID = executionPath.substring(executionPath.lastIndexOf("/") + 1);
```

Comment:
There is already simulationID, no need to add another field if it has the same information.

Reply:
Removed workflowID